app/src/main/java/org/simple/clinic/drugs/selection/custom/CustomDrugEntryEffectHandler.kt

```diff
@@ -57,7 +57,7 @@ class CustomDrugEntryEffectHandler @AssistedInject constructor(
   }
 
   private fun setSheetTitle(setSheetTitle: SetSheetTitle) {
-    val sheetTitle = constructSheetTitle(setSheetTitle.name, setSheetTitle.dosage, setSheetTitle.frequency)
+    val sheetTitle = constructSheetTitle(setSheetTitle.name, setSheetTitle.dosage.nullIfBlank(), setSheetTitle.frequency)
```

Comment:
Is there a scenario where this can be blank instead of `null` or `Unknown`? 🤔

Reply:
This is for dosage text. How it is currently implemented is if dosage doesn't have focus and there is only `mg` in the edit text then it sets it to an empty string. You can go to `CustomDrugEntryUiRenderer#showDefaultDosagePlaceholder` method to see the implementation. And  I'm using `listOfNotNull` for constructing the sheet title which led to the title having `DrugName` and a `,`. Hence I added this.